src/settings.ts

```diff
@@ -771,6 +771,9 @@ Object.freeze(ALWAYS_EMPTY_ARRAY);
 export const ALWAYS_NOOP_FUNCTION = () => {};
 Object.freeze(ALWAYS_NOOP_FUNCTION);
 
+// NOTE: flag for components that display Not Found screen or use some mitigation when no venueId or no venue exists
+export const ALLOW_NO_VENUE: boolean = true;
```

Comment:
This is unnecessary, even if we are to add dynamic flag it will be coming from the database and not the env/settings file.

Reply:
I suppose the only benefit is to use it in all of the places atm and then just change it with the database value.

Reply:
> This is unnecessary, even if we are to add dynamic flag it will be coming from the database and not the env/settings file.

It is not intended to be a dynamic flag. It was intended to be a transitional switch until all code that depends on `venue`/`venueId` existing for the `NavBar` to be visible has been merged. This switch would help those PRs turn this off in case of a problem.

Reply:
It is not intended to be a dynamic flag. It was intended to be a transitional switch until all code that depends on venue/venueId existing for the NavBar to be visible has been merged. This switch would help those PRs turn this off in case of a problem.